components/post_markdown/post_markdown.jsx

```diff
@@ -50,6 +54,10 @@ export default class PostMarkdown extends React.PureComponent {
         options: {},
     };
 
+    memoizedGetMentionKeysWithoutChannelMentions(mentionKeys) {
+        return memoizeResult(() => mentionKeys.filter((value) => !['@all', '@channel', '@here'].includes(value.key)));
+    }
```

Comment:
I think this should be in the container component and then passed in as a prop shouldnt it? i.e. `/components/post_markdown/index.js`

Reply:
I'm not sure if I quite understand what you mean.

Reply:
Look at how the memoization is done in `components/more_direct_channels/index.ts`

Reply:
Done! I made it the same way it's done there.